extensions/amp-app-banner/0.1/amp-app-banner.js

```diff
@@ -381,7 +381,7 @@ export class AmpAndroidAppBanner extends AbstractAppBanner {
 
   /** @private */
   getAndroidIntentForUrl_(appId) {
-    const canonicalUrl = documentInfoFor(this.win).canonicalUrl;
+    const canonicalUrl = documentInfoForDoc(this.element).canonicalUrl;
```

Comment:
It's kinda weird that we call this `documentInfoForDoc` but pass in an `Element`.